src/device.c

```diff
@@ -45,5 +51,8 @@ const char capabilities_str_short[NUM_CAPABILITIES]
           [CAP_EQUALIZER_PRESET]               = 'p',
           [CAP_EQUALIZER]                      = 'e',
           [CAP_MICROPHONE_MUTE_LED_BRIGHTNESS] = 't',
-          [CAP_MICROPHONE_VOLUME]              = 'o'
+          [CAP_MICROPHONE_VOLUME]              = 'o',
+          [CAP_VOLUME_LIMITER]                 = 'g'//,
+          //[CAP_BT_WHEN_POWERED_ON]             = 'h',
+          //[CAP_BT_CALL_VOLUME]                 = 'c'
```

Comment:
The `capabilities_str_short` is basically used for the deprecated short output future.

I think I will replace it with \0 for the new features, and add a check here https://github.com/Sapd/HeadsetControl/blob/f64ac48384cbb7df3878749cbe9635adefdae84c/src/output.c#L728 that it does not output something on \0

Reply:
> The `capabilities_str_short` is basically used for the deprecated short output future.
> 
> I think I will replace it with \0 for the new features, and add a check here
> 
> https://github.com/Sapd/HeadsetControl/blob/f64ac48384cbb7df3878749cbe9635adefdae84c/src/output.c#L728
> 
> that it does not output something on \0

There I've putted some random letters as a placeholder... if you want to remove them after my edits, you're free to do it.